README.md

```diff
@@ -42,6 +42,7 @@ Have any feedback or want to share your good/bad experience with Gatus? Feel fre
     - [Configuring Discord alerts](#configuring-discord-alerts)
     - [Configuring Email alerts](#configuring-email-alerts)
     - [Configuring Mattermost alerts](#configuring-mattermost-alerts)
+    - [Configuring Google chat alerts](#configuring-gchat-alerts)
```

Comment:
I know this is going to sound petty, but could you place it so it's sorted alphabetically? Makes it easier to browser the documentation 😅